extensions/wikia/Chat2/Chat.i18n.php

```diff
@@ -119,7 +119,9 @@
 	'chat-browser-is-notsupported' => '	Your browser is not supported. For the best experience, use a <a href="http://community.wikia.com/wiki/Help:Supported_browsers">newer browser</a>.',
 	'chat-checkuser-join-action' => 'Joined Chat',
 
-	'chat-status-away' => 'Away'
+	'chat-status-away' => 'Away',
+	
+	'chat-limit-reached' => 'Sorry, but your message is too long. You may only send no more than $1 characters',
```

Comment:
@gitkuba — for more information, refer to https://www.mediawiki.org/wiki/Help:System_message#Finding_messages_and_documentation